app/wizards/placements/base_step.rb

```diff
@@ -10,6 +10,16 @@ def initialize(wizard:, attributes:)
   end
 
   def to_partial_path
-    self.class.name.demodulize.underscore
+    "#{base_path}#{class_to_path(self.class)}"
+  end
+
+  private
+
+  def base_path
+    "placements/wizards/#{class_to_path(wizard.class)}/"
+  end
+
+  def class_to_path(klass)
+    klass.name.demodulize.underscore
```

Comment:
```
def to_partial_path
  "wizards/#{self.class.name.underscore}"
end
```

Instead of:
- app/views/placements/wizards/add_placement_wizard/mentors_step

How about:
- app/views/wizards/placements/add_placement_wizard/mentors_step

Would make the implementation simpler, and maybe more convention-y?